.github/workflows/codespell.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [master]
```

Comment:
I thinks these should be replaced by "main"

Reply:
```sh
$ time codespell >/dev/null

real	0m0,715s
user	0m0,679s
sys	0m0,036s
```
$

Reply:
Ooops... This was a copy/paste from another project!
`master` → `main`